consensus/mint-client/src/config.rs

```diff
@@ -247,6 +333,14 @@ pub fn load_key_from_pem(filename: &str) -> Result<Ed25519Private, String> {
         .map_err(|err| format!("Failed parsing DER from PEM file '{}': {}", filename, err))
 }
 
+pub fn parse_ed25519_signature(hex_signature: &str) -> Result<Ed25519Signature, String> {
+    let bytes = hex::decode(hex_signature)
+        .map_err(|err| format!("Failed decoding hex signature: {}", err))?;
+
+    Ed25519Signature::try_from(&bytes[..])
+        .map_err(|err| format!("Failed parsing Ed25519 signature: {}", err))
+}
+
```

Comment:
Do we want to use bytes here, or PEM/DER: https://github.com/mobilecoinfoundation/mobilecoin/blob/9450b1966cef23cedd53cac5f169110019fe2f2d/crypto/keys/src/ed25519.rs#L78

Reply:
I have no preference, and this is easy to change. @holtzman might have an opinion, as he's been playing with HSMs.

Reply:
Trezor's APIs are in Python, and it's easier to deal with it in hex or base64 rather than PEM.

Reply:
@jcape are you okay leaving it as bytes? or do you want me to switch to PEM+DER?

Reply:
If the wallets is using hex, and the signature component order---Ed25519 signatures have two components---matches that's fine.

I expect "industrial" HSMs to do PEM, which will require some massaging to get into the correct hex format.